_posts/0000-01-02-jalenS13.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Do not fear failure Not failure, but low aim is the crime in great attempts it is glorious even to fail"